themes/digital.gov/src/js/guide-menu-bar.js

```diff
@@ -0,0 +1,59 @@
+// Uses manual breakpoints because USWDS breakpoints are not accessible within JS
+const deviceBreakpoint = 480;
+const menuBarScrollOffsetMobile = 80;
+const menuBarScrollOffsetFull = 240;
+const menuBarScrollOffsetDefault = 30;
+
+const menuBar = document.querySelector(".dg-guide__menu-bar");
+const menuBarLinks = menuBar.querySelector(".dg-guide__menu-bar-links");
+const menuBarImageFull = menuBar.querySelector(
+  ".dg-guide__menu-bar-image.full"
+);
+const menuBarImageMobile = menuBar.querySelector(
+  ".dg-guide__menu-bar-image.mobile"
+);
+
```

Comment:
These selectors cause an error on homepage if `dg-guide__menu-bar` doesn't exist.

**Steps to reproduce**
1. Visit [homepage preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-new-guides-template/).
2. Open up devtools.
3. Go to console and see error.

```bash
Uncaught TypeError: Cannot read properties of null (reading 'querySelector')
    at guide-menu-bar.js:8:30
```

We should check for the top-level class and return if it doesn't exist. See `summary-box.js` for an example:
https://github.com/GSA/digitalgov.gov/blob/135245591cf61f287e9f90cfcf4f256add7ff467/themes/digital.gov/src/js/summary-box.js#L9